client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -176,9 +176,9 @@ export default function StepBusinessInfo( props: WooCommerceInstallProps ): Reac
 						<SupportCard />
 						<StyledNextButton
 							onClick={ () => {
-								dispatch( submitSignupStep( { stepName: 'business-info' } ) );
 								updateOnboardingProfile( 'completed', true );
 								save();
```

Comment:
This `updateOnboardingProfile('completed', true);` doesn't change `editedSettings` in time for the `save()` function to see it up in ln84.